### ISSUE TITLE
MacOS playing fails after updating PMS

### ISSUE THREAD

**Issue by theskillwithin** [plexinc/plex-media-player#974]
PMP logs: https://gist.github.com/theskillwithin/8cf069ebf5053c4856569491d05b41eb

### Test environment

PMS Version: Version 1.19.5.3112
PMP Version:  Version 1.15.0.1354-d31489f0
Platform: Mac

### Steps to reproduce
1.  Upgrade PMS to Version 1.19.5.3112
2. try to play video on mac desktop

### Current behavior
1. When playing on mac desktop get error
```
There was an unexpected error during playback. Please visit our forums if you continue to experience problems.
```

when playing any video.

but if going to plex.tv in browser works

**Comment by mseeley** [comment]
@theskillwithin will you please share the PMP log files themselves? The ones above are helpful but capture most of the activity only from the UI's point of view.

See: https://support.plex.tv/articles/207338748-plex-media-player-logs/

**Comment by theskillwithin** [comment]
@mseeley https://drive.google.com/drive/folders/14qmB3ZoBavEkq1DccTNLZbWGf29vo1n1?usp=sharing

**Comment by theskillwithin** [comment]
https://gist.github.com/theskillwithin/4f577f664e0ecefca52fb8e2fe23fc43

**Comment by theskillwithin** [comment]
restarting my NAS and my computer seems to have no effect.  

however something very interesting happened: 

I noticed it wasn't working on my desktop, but on my laptop it was working (same os).  but then i tried restarting my router,  and all the sudden my desktop computers PMP is working and my laptops is broken now.  so they switched magically...

**Comment by mseeley** [comment]
@theskillwithin there was an ugly bug in Plex for Windows / Plex for Mac in 1.15.0. We released 1.15.1 yesterday to resolve it. This will have addressed the issue you mentioned above in version 1.15.0.1354-d31489f0.

**Comment by theskillwithin** [comment]
@mseeley Thanks!  I can confirm after updating PMP on my laptop to new version (wasn't sure how to update but after opening and closing the app several times seemed to trigger it), that it is working now!

P.S. kinda weird I noticed when looking through logs had react errors in it, but this repo doesn't have any JS.  I'd love to contribute to the react project someday if thats public.
